db/Controllers/articles-controller.js

```diff
@@ -13,4 +13,33 @@ function getArticleByID(req, res, next){
     });
 }
 
-module.exports = getArticleByID;
\ No newline at end of file
+function getCommentsByID(req, res, next) {
+    const { article_id } = req.params;
+
+    // Promise.all([fetchArticleByID(article_id), fetchCommentsByID(article_id)])
+    // .then((comments)=>{
+
+    //     res.status(200).send(comments)
+    // })
+    // .catch((err)=>{
+    //     next(err);
+    // });
+
+    fetchArticleByID(article_id)
+    .then(()=>{
+        fetchCommentsByID(article_id)
+        .then((comments)=>{
+            console.log(comments);
+            res.status(200).send(comments)
+        })
+        .catch((err)=>{
+            next(err);
```

Comment:
Your controller only needs one catch block. Your model returns a promise chain and if an error is thrown inside of it, then the catch block for your controller will catch it and pass it on to the correct error handling middleware